public/countries.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<link rel="icon" href="assets/image/favicon.ico">
+
+		<title>OpenFisca - Available country packages</title>
+
+		<link rel="stylesheet" type="text/css" href="assets/css/main.css">
+	</head>
+
+	<body>
+		<header>
+			<a href="./index.html">
+				<img id="logo" src="assets/image/logo-openfisca.svg" alt="OpenFisca">
+			</a>
+
+			<h1>Available country packages</h1>
+
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		</header>
+
+		<section>
+			<h2>These countries have their tax and benefit system model.</h2>
+			<div><p>
+				Country Packages are the basic modules of OpenFisca. They define the Parameters, Entities and Variables of a country.
+			</p></div>
+		</section>
+
+		<section id="fr">
+			<div>
+				<h3>France</h3>
+				<div id="gradientfr"></div>
+				<p>
+					See the source code of this project on <a href="https://github.com/openfisca/openfisca-france">github</a>.
+				</p>
+			</div>
+			<a href="https://legislation.openfisca.fr" class="cta">Explore</a>
+		</section>
+
+		<section id="tn">
+			<div>
+				<h3>Tunisia</h3>
+				<div id="gradienttn"></div>
+				<p>
+					See the source code of this project on <a href="https://github.com/openfisca/openfisca-tunisia">github</a>.
+				</p>
+			</div>
+			<a href="https://legislation.openfisca.tn" class="cta">Explore</a>
+		</section>
+
+		<section id="sn">
+			<div>
+				<h3>Senegal</h3>
+				<div id="gradientsn"></div>
+				<p>
+					See the source code of this project on <a href="https://github.com/openfisca/senegal">github</a>.
+				</p>
+			</div>
+		</section>
+
+		<section>
+			<div>
+				<h3>Yours is missing?</h3>
+			</div>
+			<a href="https://doc.openfisca.fr/coding-the-legislation/bootstrapping_a_new_country_package.html" class="cta">Start modeling!</a>
```

Comment:
The button should have more padding (cf "start modeling") --> applies to all ctas